File: 백준/Bronze/34308. Abby's Absolutes/Abby's Absolutes.cc
#include <iostream>
#include <vector>
using namespace std;
int main(void) {
    int n,k,l;
    cin>>n>>k;
    l=n/2+(n%2==0?0:1);
    vector<int> arr(k);
    for(int i=0;i<k;i++) {
        cin>>arr[i];
        cout<<(arr[i]<=l?1:n)<<' ';
    }
    return 0;
}

File: 백준/Bronze/34308. Abby's Absolutes/Abby's Absolutes.java
import java.util.Scanner;
import java.util.Arrays;
public class Main {
    public static void main(String[] args) {
        int n,k,l;
        Scanner scan = new Scanner(System.in);
        n=scan.nextInt();
        k=scan.nextInt();
        l=n/2+(n%2==0?0:1);
        scan.nextLine();
        int[] arr=Arrays.stream(scan.nextLine().split(" "))
        .mapToInt(Integer::parseInt).toArray();
        for(int i=0;i<k;i++) {
            System.out.print((arr[i]<=l?1:n)+" ");
        }
    }
}